docusaurus.config.js

```diff
@@ -1,39 +1,39 @@
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require("prism-react-renderer/themes/github");
```

Comment:
Please turn off auto-formatting on your PR so you only change the lines that actually needs to be fixed.